bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingAddOp.java

```diff
@@ -344,7 +347,11 @@ void submitCallback(final int rc) {
         } else {
             addOpLogger.registerSuccessfulEvent(latencyNanos, TimeUnit.NANOSECONDS);
         }
-        cb.addComplete(rc, lh, entryId, ctx);
+        if (lh instanceof LedgerHandleAdv) {
```

Comment:
This instanceof is unnecessary. addCompleteAdv calls addComplete by default anyhow.

Reply:
Yeah, will fix.